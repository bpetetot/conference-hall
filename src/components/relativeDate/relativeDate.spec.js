/* eslint-env jest */
import snap from 'tests/snapshot'
import Titlebar from './index.js'

jest.mock('date-fns/distance_in_words_to_now', () => () => '2 days ago')
jest.mock('../iconLabel', () => 'IconLabel')

const snapshot = props => snap(Titlebar)({ ...props })

describe('components/titlebar', () => {
  it('should not render if no date', snapshot({}))
  it('should not render if no date', snapshot({ date: new Date() }))
})
