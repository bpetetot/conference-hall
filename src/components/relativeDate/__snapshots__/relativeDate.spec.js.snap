// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/titlebar should not render if no date 1`] = `null`;

exports[`components/titlebar should not render if no date 2`] = `
<IconLabel
  icon="fa fa-clock-o"
  label="2 days ago"
/>
`;
